Basic-Car-Maintenance/Documentation.docc/Documentation.md

```diff
@@ -16,3 +16,76 @@ Icons have the following colors:
 * Blue #253AA7
 * Black #000000
 * Orange #F78616
+
+## Firestore Collections
+
+**_alerts_**  
+The alerts collection contains system level alerts that will be visable to all users
```

Comment:
```
The alerts collection contains system level alerts that will be visable to all users
```

More so than this, can you also note that they will only be shown to the user once